internal/os/os_exec_commands.go

```diff
@@ -21,13 +21,20 @@ type OsExecCommands interface {
 	UpdateUrlInEdgeRemote(newURL string, remoteFileName string) error
 }
 
-type osExecCommands struct {}
+type osExecCommands struct{}
 
-func NewOsExecCommands() OsExecCommands{
+func NewOsExecCommands() OsExecCommands {
 	return &osExecCommands{}
 }
 
-func (o *osExecCommands) RpmOstreeStatus() ([]byte, error){
+func (o *osExecCommands) IsRpmOstreeAvailable() bool {
+	cmd := exec.Command("which", "rpm-ostree")
```

Comment:
is which always present? I remember having an issue similar to this, but I do not recall where :\

Reply:
Done - replaced with `rpm-ostree status`.